remix-tests/src/runTestFiles.ts

```diff
@@ -8,8 +8,17 @@ import Web3 = require('web3')
 import { compileFileOrFiles } from './compiler'
 import { deployAll } from './deployer'
 
+/**
+ * @dev run test contract files (used for CLI)
+ * @param filepath Path of file
+ * @param isDirectory True, if path is a directory
+ * @param web3 Web3
+ * @param opts Options
+ */
+
 export function runTestFiles(filepath: string, isDirectory: boolean, web3: Web3, opts?: object) {
     opts = opts || {}
+    let sourceASTs: any = {}
```

Comment:
This is interesting. Here you create an empty object that you _mutate_ later on. But you do not _reassign_ the value. Which means that you can use `const` here ;)